### PR TITLE
Corrected minor typos

### PR DESCRIPTION
**Pull request by SinghRajenM**
In parameters.h, only extra semicolons are removed.

### CODE REVIEW

File: PowerEditor/resources/nppPluginList.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
 			"folder-name": "mime-tools",
 			"display-name": "Mime tools",
 			"repository": "https://github/npp_plugins/mimetools/",
-			"description": "MIME plugin for Notepad++ implements several main functionnalities defined in MIME (Multipurpose Internet Mail Extensions).",
```
Comment (chcg) [inline PowerEditor/resources/nppPluginList.json]: Already correct at https://github.com/notepad-plus-plus/nppPluginList/blob/master/src/pl.x86.json

Comment (SinghRajenM) [reply PowerEditor/resources/nppPluginList.json]: @chcg Do we need this file in npp repository? I don't think so.

Comment (chcg) [reply PowerEditor/resources/nppPluginList.json]: @SinghRajenM You are right. This file is not needed any longer in the npp repo itself and is outdated version 1.0 anyway.

Comment (donho) [reply PowerEditor/resources/nppPluginList.json]: @SinghRajenM Yes, this file should be removed.

File: PowerEditor/src/Parameters.h [hunk 2]
```diff
@@ -1280,7 +1280,7 @@ const int RECENTFILES_SHOWONLYFILENAME = 0;
 class NppParameters final
```
Comment (donho) [inline PowerEditor/src/Parameters.h]: Please keep `Parameters.h` as it is.

**Comment by SinghRajenM** [comment]
@donho review comments have been accommodated.

**Pull request closed without merging**